components/channel_header_mobile/collapse_lhs_button/collapse_lhs_button.tsx

```diff
@@ -2,24 +2,27 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
 
 import NotifyCounts from 'components/notify_counts';
 import MenuIcon from 'components/widgets/icons/menu_icon';
 
-const CollapseLhsButton = ({
-    actions: {
-        toggleLhs,
-    },
-}) => (
+type Actions = {
+    toggleLhs: (e?:React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+}
+
+type Props = {
+    actions: Actions
```

Comment:
Are we able to do this? I feel since it's just one function, we don't need a whole different type.
```
actions: { 
         toggleLhs: (e?:React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
}
```

That way we don't need an entire different type of `Action`. It's a non-blocking change but I'd prefer if we didn't need the whole different type. Thanks!

Reply:
I've made the changes.. :)